mrp_production_service/models/mrp_production.py

```diff
@@ -9,13 +9,13 @@ class MrpProduction(models.Model):
     _inherit = "mrp.production"
 
     @api.model
-    def _prepare_service_procurement_values(self, production):
-        location = production.location_src_id
+    def _prepare_service_procurement_values(self):
+        location = self.location_src_id
```

Comment:
then this is not anymore api.model but api.multi

Reply:
Done, is it right this way?